CMakeLists.txt

```diff
@@ -217,18 +223,20 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/softwarecontainer.pc DESTINATION ${CMA
 #
 # General configuration file
 #
+# The *.h.in file is being made obsolete, TODO: Remove this
 configure_file(softwarecontainer-config.h.in softwarecontainer-config.h @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/softwarecontainer-config.h DESTINATION ${CMAKE_INSTALL_FULL_LIBDIR}/softwarecontainer)
 
+set(SERVICE_MANIFEST_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/softwarecontainer/service-manifest.d/")
+set(DEFAULT_SERVICE_MANIFEST_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/softwarecontainer/service-manifest.default.d/")
+
 #
 # Add all sub-projects
 #
-
 add_subdirectory(common)
 add_subdirectory(libsoftwarecontainer)
 
 add_subdirectory(agent)
-add_subdirectory(libsoftwarecontaineragent)
```

Comment:
Yes I know. It's just that I have to fix tests and build errors etc., and I know we are to remove it. I guess this will go away during a rebase soon.